src/wp-includes/option.php

```diff
@@ -931,7 +934,7 @@ function update_option( $option, $value, $autoload = null ) {
  *                                to not autoload them, by using 'no'|false. Default 'yes'.
  * @return bool True if the option was added, false otherwise.
```

Comment:
Nit-pick: The documentation of all these parameter docs needs to be fixed to be consistent.

Reply:
tidied